.github/workflows/ghcr_build.yml

```diff
@@ -0,0 +1,85 @@
+name: Build GHCR images
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request_target:
+    branches: [ "master" ]
+
+env:
+  REGISTRY_IMAGE: ghcr.io/${{ github.repository }}
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  teslamate_build:
+    name: Build images
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - platform: "linux/amd64"
+            runs_on: "ubuntu-latest"
+            cache_id: amd64
+#          - platform: "linux/arm/v7"
```

Comment:
please enable all platforms

Reply:
done. forkers who want to build will have to use the buildjet service